specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2019-06-01/managedClusters.json

```diff
@@ -1417,8 +1417,7 @@
         }
       },
       "required": [
-        "vmSize",
-        "count"
+        "vmSize"
```

Comment:
vmSize is also not required when updating an existing agent pool

Reply:
and it is a immutable value, we will throw error if customer provides different vmSize with the one in our record.

Reply:
good point - I'll update this 👍

Reply:
pushed 👍